.github/CODEOWNERS

```diff
@@ -5,6 +5,7 @@ script/uplift.py @bsclifton
 vendor/brave-ios/ @kylehickinson
 /README.md @bsclifton
 Jenkinsfile @mihaiplesa
+/ts* @petemill
```

Comment:
what will be handled with this one?

Reply:
are you targeting TS json files in root folder? If so I think it should be like `/ts*.json` as `/ts*` could include folder or something else as well if I am not mistaken